airflow/models.py

```diff
@@ -7,9 +7,9 @@
 # to you under the Apache License, Version 2.0 (the
 # "License"); you may not use this file except in compliance
 # with the License.  You may obtain a copy of the License at
-# 
+#
```

Comment:
trailing spaces trim was done by PyCharm automatically.. not sure how to disable whitespace normalization. hope that's ok)

Reply:
This is not a problem